independent-projects/resteasy-reactive/pom.xml

```diff
@@ -59,7 +59,7 @@
         <jackson.version>2.13.1</jackson.version>
         <smallrye-stork.version>1.0.0.Alpha7</smallrye-stork.version>
         <jakarta.validation-api.version>2.0.2</jakarta.validation-api.version>
-        <yasson.version>2.0.2</yasson.version>
+        <yasson.version>2.0.4</yasson.version>
```

Comment:
Hmmmm, we have an issue here. We should not use 2.0 as it's using the `jakarta.*` packages. We need the versions to be consistent and aligned so we should use `1.0.11` everywhere.

Reply:
well, I've been suprised to see this 2.x here as well but since it has been there before and since resteasy-reactive is a standalone-module I've expected it to work already.

Reply:
I just double checked the 2.6.x branch (since I thought that maybe main is already transition to jakarta-packages), but 2.6 also already uses yasson 2.0.2

Reply:
Well, it's an independent module but in the end it's integrated into Quarkus and uses the 1.0 version enforced by the BOM. So I think we need to fix it. Could you try adjusting it? I'll approve the workflow run and then we can see how CI reacts.

Reply:
Sure, I've already tested the downgrade with `mvnw -Dquickly` and it compiled at least.